boilerplate/app/models/AuthenticationStore.ts

```diff
@@ -4,8 +4,8 @@ export const AuthenticationStoreModel = types
   .model("AuthenticationStore")
   .props({
     authToken: types.maybe(types.string),
-    authEmail: types.optional(types.string, ""),
-    authPassword: types.optional(types.string, ""),
+    authEmail: types.optional(types.string, "ignite@infinite.red"),
+    authPassword: types.optional(types.string, "ign1teIsAwes0m3"),
```

Comment:
I'm almost wondering if we should move the default values to the LoginScreen's useLayoutEffect (or similar). The reason being - this implementation is great to demo the "default value" feature of MST, but it's not a realistic use-case for the email/pass. If moved to the LoginScreen, we could add some comments like "this is where you could fetch the user/pass from keychain" or whereever. 

Anyways, not a huge deal, just throwin out ideas for a clearer intent.

Reply:
@yulolimum Hmm, yeah i agree. Let me try something like that.